inventories/seapath_standalone_definition_example.yml

```diff
@@ -4,46 +4,61 @@ all:
-        # Optional list variable to change apt repositories. All repositories will be overrides.
-        # If defined as empty list [], all apt repositories will be removed
-        apt_repo:
-            - http://ftp.fr.debian.org/debian bullseye main contrib non-free
-            - http://security.debian.org/debian-security bullseye-security main contrib non-free
-            - https://download.docker.com/linux/debian bullseye stable
-            - https://artifacts.elastic.co/packages/8.x/apt stable main
+
+                    # Realtime/CPUs cgroups isolation configuration
+                    workqueuemask: "1001" #workqueue mask, here it mean 0 and 12 are the only allowed cpus
+                    cpusystem: "0,12" # CPUs reserves for system
+                    cpuuser: "1,13" # CPUs reserves for user applications
+                    cpumachines: "2-11,14-23" # CPUs reserves for VMs
+                    cpumachinesrt: "2-5,14-17" # CPUs reserves for VMs realtime
+                    cpumachinesnort: "6-11,18-23" # CPUs reserves for VMs non realtime
+                    cpuovs: "23" # CPUs reserves for OVS
+
+                    # Grub password
+                    # The password hash generated with "grub-mkpasswd-pbkdf2 -c 65536 -s 256 -l 64", in this example the pass is "toto"
+                    grub_password: grub.pbkdf2.sha512.65536.E291D66AEEB3C22BD6B019C5C3587A3094AE93D61E20D134EC6324925AE5045DCA61EF30B3BD04B4D6F7360B9C9B242AA68B1643CCB269C53658EC959B5964ADB9C9D5FAA280A291D8F95E3F255254A4119A2431AFE797F1949EE4FBBC4C74281C550C83DAED26C254224061BEFCEEBF8091A8D1BE406EBB3A3E8A519E36B4FE161BE191D407193E5DFEBCC09F8822B4060EA9CD1E6B8677D40D32826EF025CA494BCD209032F7CF2A4A2E74717E6D17E87A62AAA93E458C96F983E69FBFC4FD602403988EAF5AADCA4B5B145B0F6C6FFB53F55CD6C56C15C17B2F8A5B3A214F3140470566597760D9388084AE978DB5C0EBF7C868A855DB38DACA47A010417A.FDA0C3188FAC87FFA04D862AC9B1020F29FEEA3B9590BE534330D8C7CAB71444CBF527E39D7DAE640545139202B9CA77822CCE21AB1134110F6AF3EFD793E848
+
+                    # Optional list variable to change apt repositories. All repositories will be overrides.
+                    # If defined as empty list [], all apt repositories will be removed
+                    apt_repo:
+                        - http://ftp.fr.debian.org/debian bullseye main contrib non-free
+                        - http://security.debian.org/debian-security bullseye-security main contrib non-free
+                        - http://ftp.fr.debian.org/debian bullseye-backports main contrib non-free
```

Comment:
why add this line ?

Reply:
I reuse the apt_repo from the previous inventory.
The kernel we use came from bullseye-backports. We need this repo if we want update it.